state-sync/src/request_manager.rs

```diff
@@ -432,8 +445,8 @@ impl RequestManager {
     }
 
     /// Checks whether the request sent with known_version = `version` has timed out
-    /// Returns true if such a request timed out or does not exist, else false
-    pub fn check_timeout(&mut self, version: u64) -> Result<bool> {
+    /// Returns true if such a request timed out (or does not exist), else false.
+    pub fn check_request_timeout(&mut self, version: u64) -> Result<bool> {
```

Comment:
perhaps `has_request_timedout` might be better, but I don't think we have a naming scheme for that / I'm used to writing stuff in other languages

Reply:
Yeah, this is not super clear. Plus this entire method still needs to be refactored... Perhaps `has_request_timed_out`?